src/DynamoCore/Graph/Workspaces/LayoutExtensions.cs

```diff
@@ -426,6 +433,46 @@ private static void SaveLayoutGraph(this WorkspaceModel workspace, List<GraphLay
                     node.ReportPosition();
                     workspace.HasUnsavedChanges = true;
 
+                    double noteOffset = -n.NotesHeight;
+                    foreach (NoteModel note in n.LinkedNotes)
+                    {
+                        if (note.IsSelected || DynamoSelection.Instance.Selection.Count == 0)
+                        {
+                            note.X = node.X;
+                            note.Y = node.Y + noteOffset;
+                            noteOffset += note.Height + GraphLayout.Graph.VerticalNoteDistance;
+                            note.ReportPosition();
+                        }
+                    }
+                }
+            }
+        }
+        /// <summary>
+        /// This method pushes changes from the GraphLayout.Graph objects
+        /// back to the workspace models, but only for nodes placed by NodeAutocomplete.
+        /// </summary>
```

Comment:
Both versions of the function are missing comments for input params, do you mind adding them back?